cptoml.py

```diff
@@ -1,44 +1,55 @@
-def _prepareline(line) -> str:
+class InvalidTOMLSyntax(Exception):
+    """Custom class for bad syntax."""
+    pass
+
+def _prepareline(line: str) -> str:
     """
     Remove comments from a line buffer.
     Also removes misc chars.
     """
-    if (
-        (line.rfind("#") != -1)
-        and (
-            line.find("'") != -1
-            and line.find("'") != line.rfind("'")
-            and (line.find("'") > line.rfind("#") or line.rfind("'") < line.rfind("#"))
-        )
-        and (
-            line.find('"') != -1
-            and line.find('"') != line.rfind('"')
-            and (line.find('"') > line.rfind("#") or line.rfind('"') < line.rfind("#"))
-        )
-    ):  # Without the -1 check eats a char
-        line = line[: line.rfind("#")]
-    while line.endswith(" ") or line.endswith("\n"):
-        line = line[:-1]
-    while line.startswith(" "):
-        line = line[1:]
-    return line
-
-
-def _dataformat(data):
-    """
-    Prepares the data into a list.
-    """
-    data = data.split("\n")
-    for i in range(len(data) - 1, 0, -1):  # Go in reverse as to not break the index
-        if data[i].isspace() or not data[i]:  # "" == False
-            data.pop(i)
-    return data
 
+    # strip takes a string as argument it will remove any char on it
+    # from the head and tail of input string
+    _CHARS_TO_REMOVE = " \n"
```

Comment:
I mean, the function only receives one line at a time. So there will be no `\n`?

Reply:
That's what i thought, but since you were looking for \n too...